### PR TITLE
Rewrite test based on enzyme

### PR DESCRIPTION
**Pull request by edgarmueller**

### CODE REVIEW

File: packages/material/test/renderers/MaterialArrayLayout.test.tsx [hunk 1]
```diff
@@ -154,49 +157,52 @@ describe('Material array layout', () => {
...
       </Provider>
-    ) as React.Component<any, any, any>;
+    );
 
-    const controls = TestUtils.scryRenderedDOMComponentsWithTag(tree, 'input');
+    const controls = wrapper.find('input');
     // 2 data entries with each having 2 controls
-    expect(controls.length).toBe(4);
+    expect(controls).toHaveLength(4);
+    wrapper.unmount();
```
Comment (eneufeld) [inline packages/material/test/renderers/MaterialArrayLayout.test.tsx]: why not use `afterEach(() => wrapper.unmount());` once as in the arraycontrol test?

File: packages/material/test/renderers/MaterialDateTimeControl.test.tsx [hunk 2]
```diff
@@ -195,195 +143,194 @@ describe('Material date time control', () => {
...
-    ) as React.Component<any, any, any>;
-    const input = TestUtils.findRenderedDOMComponentWithTag(tree, 'input') as HTMLInputElement;
-    // there is only input id at the moment
-    expect(input.id).toBe('#/properties/foo-input');
+      </Provider>
+    );
+    const input = wrapper.find('input').first();
+    expect(input.props().autoFocus).toBeTruthy();
   });
+  //
```
Comment (eneufeld) [inline packages/material/test/renderers/MaterialDateTimeControl.test.tsx]: why is everything commented out?

File: packages/material/test/renderers/MaterialRadioGroupControl.test.tsx [hunk 3]
```diff
@@ -78,41 +79,37 @@ const initJsonFormsStore = (testData: any, testSchema: JsonSchema, testUiSchema:
...
-    const radioButtons = _.filter(inputs, i => i.type === 'radio');
+    const radioButtons = wrapper.find('input[type="radio"]');
+    const currentlyChecked = wrapper.find('input[type="radio"][checked=true]');
     expect(radioButtons.length).toBe(4);
-    // make sure one option is selected and it is "D"
-    const currentlyChecked = _.filter(radioButtons, radio => radio.checked);
-    expect(currentlyChecked.length).toBe(1);
-    expect(currentlyChecked[0].value).toBe('D');
+    expect(currentlyChecked.first().props().value).toBe('D');
+    wrapper.unmount();
```
Comment (eneufeld) [inline packages/material/test/renderers/MaterialRadioGroupControl.test.tsx]: I would suggest to use `afterEach(() => wrapper.unmount());` instead

File: packages/material/test/renderers/MaterialSliderControl.test.tsx [hunk 4]
```diff
@@ -30,30 +30,31 @@ import {
...
-      default: 6
-    },
-  },
-};
+import Enzyme, { mount, ReactWrapper } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+// const data = {'foo': 5};
```
Comment (eneufeld) [inline packages/material/test/renderers/MaterialSliderControl.test.tsx]: if this is not needed anymore, we should delete this

File: packages/material/test/renderers/MaterialSliderControl.test.tsx [hunk 5]
```diff
@@ -242,156 +240,155 @@ describe('Material slider control', () => {
...
-      </Provider>,
-      container
-    ) as React.Component<any, any, any>;
-    const divs = TestUtils.scryRenderedDOMComponentsWithTag(tree, 'div') as HTMLElement[];
-    // id
-    expect(divs.find(d => d.id === '#/properties/foo')).toBeDefined();
-    // input id
-    expect(divs.find(d => d.id === '#/properties/foo-input')).toBeDefined();
-  });
+  // it('should update via action', () => {
```
Comment (eneufeld) [inline packages/material/test/renderers/MaterialSliderControl.test.tsx]: why is it commented out?

**Comment by edgarmueller** [comment]
Thanks for the review. It got a bit late and forgot to un-comment those tests. Those should be fine now.

File: packages/material/test/renderers/MaterialObjectControl.test.tsx [hunk 6]
```diff
@@ -131,78 +132,70 @@ describe('Material object renderer tester', () => {
...
-      </Provider>,
-      container
-    ) as React.Component<any, any, any>;
-    const input = TestUtils.scryRenderedDOMComponentsWithTag(tree, 'input') as HTMLInputElement[];
-    expect(input.length).toBe(0);
+      </Provider>
+    );
+    const inputs = wrapper.find('input');
+    expect(inputs.length).toBe(0);
+    wrapper.unmount();
```
Comment (eneufeld) [inline packages/material/test/renderers/MaterialObjectControl.test.tsx]: I would suggest to use `afterEach(() => wrapper.unmount());`

**Pull request merged**